src/test/scala/com/rxthings/di/ExampleOfTestingSpec.scala

```diff
@@ -19,6 +19,7 @@ class ExampleOfTestingSpec extends InjectSpec with Matchers {
 }
 
 class InjectedActor extends NopActor {
+  import context._
```

Comment:
>which is in line with other Akka projects, e.g. Akka Streams and Akka Http

Yeah good point.  I have to bring in a system for everything else, not having to here was a smell.